validation/scripts/Validation_dnInverse.Rev

```diff
@@ -0,0 +1,52 @@
+################################################################################
+#
+# RevBayes Validation Test: dnInverse
+#
+# Model: Various: these tests validate the behaviour of dnInverse()
+#
+#
+# authors: Martin R. Smith, following Validation_Exponential.Rev
+#
+################################################################################
+
+seed(42)
```

Comment:
the idea of validation tests (as opposed to the integration tests from the tests/ folder) is that they are supposed to check that the MCMC converges to the right distribution
so they're not supposed to rely on a seed

Reply:
Gotcha.  I wondered how these differed from tests. I've removed this file.